src/tiling/tile_restoration_state.rs

```diff
@@ -225,18 +225,21 @@ macro_rules! tile_restoration_plane_common {
       pub fn restoration_unit_last_sb_for_rdo<T: Pixel>(
         &self,
         fi: &FrameInvariants<T>,
-        sbo: TileSuperBlockOffset,
+        global_sbo: PlaneSuperBlockOffset,
```

Comment:
Does 'global' mean the sbo in the frame, instead of the tile, correct?

Reply:
yes.

Reply:
I might have called it `frame_sbo` but not really a dealbreaker

Reply:
> I might have called it `frame_sbo` but not really a dealbreaker

Yes, we will write a patch to rename it when monty is back to seep.... *_*